pyUltroid/functions/helper.py

```diff
@@ -441,13 +441,13 @@ async def progress(current, total, event, start, type_of_ps, file_name=None):
     now = time.time()
     if No_Flood.get(event.chat_id):
         if No_Flood[event.chat_id].get(event.id):
-            if (now - No_Flood[event.chat_id][event.id]) < 1.1:
+            if (now - No_Flood[event.chat_id][event.id]) < 3:
                 return
         else:
             No_Flood[event.chat_id].update({event.id: now})
     else:
         No_Flood.update({event.chat_id: {event.id: now}})
```

Comment:
Don't you think time has elapsed in there statements...?

Reply:
yeah obv, however the time difference will be very small and can be ignored, i think

Reply:
> yeah obv, however the time difference will be very small and can be ignored, i think

That's the chutiyaest statement ever! What the **FUCK** you mean "`it can be ignored" ?
Every nano second matters dude...!

Reply:
"3" not need ig